client/README.md

```diff
@@ -0,0 +1,110 @@
+# Frontend
+
+## How to set up your project?
+
+-   Install node js
+-   Install npm version 6 `npm install -g npm@6.14.11`
+-   Follow the workflow for this project (Fork, run `npm install` in client folder and finally `npm start`)
+
+Please refer to the [Environment Setup](https://github.com/SE701Group2/daily-focus/wiki/Environment-setup) Link in table of content for more information
+
+## High-level architecture
+
+FOCUS is built using React v16.14.0. This is used for compatibility with the Enzyme testing library
+
+-   [Material UI](https://material-ui.com/) (core and icon) and [Font Awesome](https://fontawesome.com/) are used for the application look and feel.
+-   [use-persisted-state](https://github.com/donavon/use-persisted-state) library is used to store data when an API endpoint is not yet available.
+
+## Coding,Styling and File Struture
+
+`style.module.scss` is used for styling purposes.
+
+### File structure of the app:
+
+```
+src
+--- App
+--- components
+--- images
+--- pages
+--- index.js
+--- style.css
+```
+
+## App
+
+`App` component defines the app routes:  
+`/` and `/signup` paths show the `LandingPage`  
+`/login` path shows the `LoginPage`  
+`/home` path shows the `HomePage`
+
+## Pages
+
+`LandingPage` is where the user can signup (_note, not currently linked to backend so redirects to home page_) or redirect to the login page  
+`LoginPage` is where the user can login with their credentials  
+`HomePage` shows the main application contents
+
+## Components
+
+### Component Structure
+
+```
+MyComponent/
+--- SubComponent/
+--- --- __snapshots__
+--- --- index.js
+--- --- style.module.scss
+--- --- SubComponent.test.js
+--- ...
+--- index.js
+--- style.module.scss
+--- MyComponent.test.js
+```
+
+### Structural Components
+
+For this project, the components we have so far are:
+
+-   Header
+-   Body
+-   Core widget Components
+    -   CalendarWidget
+    -   To Do List
+    -   PlantButton
+    -   Timer
+    -   Weather
+-   Other components:
+    -   DateTime
+    -   Search Bar
+    -   SelectWidgetModal
+
+Please refer to the [Front End](https://github.com/SE701Group2/daily-focus/wiki/Front-End) under Architecture heading in the Wiki for more information about each individual components and how each of them are set up.
```

Comment:
Everything from line 18 through 81 does not belong in a README. A README's purpose is not to document the entire application structure, but to introduce and explain the project at a high level. Think, _if I am brand new to this repository, what do I need to know to get up and running?_ Not everyone is looking to contribute new components; some people just want to get a local version running on their computer. People who want to know architectural details should be directed to read the wiki.

For guidelines on what's needed in a good README, check out [Makeareadme](https://www.makeareadme.com/). Apologies, I should've sent you this website when you started this task.